### PR TITLE
Search: track total results

### PR DESCRIPTION
**Pull request by stsewd**
Closes https://github.com/readthedocs/readthedocs.org/issues/7047

### CODE REVIEW

File: readthedocs/search/models.py [hunk 1]
```diff
@@ -32,6 +32,12 @@ class SearchQuery(TimeStampedModel):
         _('Query'),
         max_length=4092,
     )
+    total_results = models.IntegerField(
+        _('Total results'),
+        default=0,
+        # TODO: to avoid downtime, remove later.
+        null=True,
```
Comment (stsewd) [inline readthedocs/search/models.py:39]: If it's ok to not save some search queries during deploy I can just remove this.

File: readthedocs/projects/views/private.py [hunk 2]
```diff
@@ -1040,7 +1040,7 @@ def _search_analytics_csv_data(self):
                 created__date__lte=now,
             )
             .order_by('-created')
-            .values_list('created', 'query')
+            .values_list('created', 'query', 'total_results')
```
Comment (ericholscher) [inline readthedocs/projects/views/private.py:1043]: We should likely show these in the user-facing table, as well as in the CSV.

**Comment by stsewd** [comment]
@ericholscher are we ok showing 0 results for old queries? Looks like integerfield treats None as 0, so I can't check for None for the old queries.

Screenshot_2020-05-14 Search Analytics Read the Docs

**Comment by ericholscher** [comment]
@stsewd yea I think thats fine for now. It will slowly fill in.

**Pull request merged**